### ISSUE TITLE
Example Gallery of feedstocks

### ISSUE THREAD

**Issue by SammyAgrawal** [leap-stc/LEAP_template_feedstock#54]
Would be great to integrate a bunch of successful use cases of feedstocks into the LEAP docs

**Comment by norlandrhagen** [comment]
Good idea! Might make it easier for people to create feedstocks. There are quite a few on [staged-recipes](https://github.com/pangeo-forge/staged-recipes), but some of them are pre-beam refactor and are missing the LEAP specific utilities. 

Also, the feedstocks are listed in the [catalog](https://catalog.leap.carbonplan.org/), maybe we could have a page or view that shows all the example feedstocks in a gallery?

**Comment by SammyAgrawal** [comment]
Hmm.. yeah or maybe as simple as having a section in the docs with individual links to these feedstocks? I think highlighting existing content could do just as much as creating new content. Every time I look closer, the catalog actually has some link to something I didn't realize was there the first time. And people might not know what feedstock means?

**Comment by jbusecke** [comment]
Good suggestions. I personally would like to avoid creating another page to highlight feedstocks, but it is well worth describing what a feedstock is and explaining the catalog UI in more detail!
